app/components/modal-confirm-deactivate/template.hbs

```diff
@@ -0,0 +1,14 @@
+<h4>Are you sure you want to deactivate:</h4>
```

Comment:
Deactivate is fine for environment, but this should probably say "stop services for:" on a stack (environment).